### ISSUE TITLE
Allow submissions through the website

### ISSUE THREAD

**Issue by Productive2** [techlore/plexus#63]
- It would lower the entry bar (no need for GH account or to open an email client and write a mail to a person)
- Some extra data could be automatically collected (linking the playstore link of the app could allow for the automatic scraping of package name, version number)
- Some data is easier to collect (month and year of testing, category of the app (perhaps in a friendly drop down menu)
- The submissions could be reviewed manually in order to avoid spam, and a privacy friendly captcha could keep bots away. This does not add too much workload, since all email or github PR submissions are reviewed manually anyways

**Comment by techlore-official** [comment]
This is already on the radar to implement-check the future plans within the readme to see them all. The goal is to jump straight into a Plexus App as this is an even easier method of submission which can be almost entirely automated. 

A website may go along well with this though so going to leave this open to see if someone would like to implement that on the site. That’s out of my experience level

**Comment by tomkonidas** [comment]
* Yes, it might lower the entry, but the site is built off PRs and that csv file. The way the site works is thatv after every PR it generates the site to raw html/css/js for fats load times instead of always querying a database when you want to go on a page.
* We cannot fully link to the playstore app via the website, as not everyone uses the same store and that is assuming they have a public API we can call to check. 
* We are working on adding more data such as categories.
* We have to review all the submissions in any way and this way through gh it could be easier to communicate. What if someone from the site sends something and we need to further validate. We cannot talk with that person per say.

We could eventually move the CSV into an actual database but then, not anyone can just got and submit a PR to put in the CSV file which could make it harder.

**Comment by pwbiuve** [comment]
I don't even know how to make a pull request. When I press new pull request it doesn't allow me to do anything. NOOB.

I think this system (pull requests) doesn't work for normal users. 

Trying to submit MANY apps - been using microg without play services for years.

**Comment by 4n0nym0u53** [comment]
> I don't even know how to make a pull request. When I press new pull request it doesn't allow me to do anything. NOOB.
> 
> I think this system (pull requests) doesn't work for normal users.
> 
> Trying to submit MANY apps - been using microg without play services for years.

@pwbiuve you need to first fork the project, add your contributions, and then create a pull request from your forked version.

**Comment by henryistaken** [comment]
Just to push an update, the Plexus application has been completed (https://github.com/techlore/Plexus-app) which is now where we will be handling contributions. The next step of the app is figuring out submissions within the app itself, and we need help with this! It's unlikely we'll do submissions through the website since the final goal is easy submissions for the end-user, and the app automatically scans for all apps on the user's device to keep this as simple as possible. 

TLDR: Easy submissions will be handled on the app, not the site. And we need help with this, so definitely shoot over ideas on the new repo if you're able to help. Thanks everyone!
